[Jenkins CI] Broken R unit test : test_img_seg

@thirdwing  

One of the R unit tests is failing in the Jenkins CI.
Looks like the issue is due to the installation of package failing.

Master builds are failing. Can you please take a look at the issue?

Build Failure Log Link -
https://builds.apache.org/blue/organizations/jenkins/incubator-mxnet/detail/master/276/pipeline

Error Output:

Failed -------------------------------------------------------------------------
1. Error: UNET (@test_img_seg.R#93) --------------------------------------------
trying to use CRAN without setting a mirror
1: install.packages(new.packages) at R-package/tests/testthat/test_img_seg.R:93
2: grep("^file:", contriburl)
3: contrib.url(repos, type)
4: stop("trying to use CRAN without setting a mirror")

DONE ===========================================================================
Error: Test failures
Execution halted
make: *** [rpkgtest] Error 1
script returned exit code 2

--- a/R-package/tests/testthat/test_img_seg.R
+++ b/R-package/tests/testthat/test_img_seg.R
@@ -90,7 +90,7 @@ context("Image segmentation")
 test_that("UNET", {
   list.of.packages <- c("imager")
   new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
-  if(length(new.packages)) install.packages(new.packages)
+  if(length(new.packages)) install.packages(new.packages, repos = "https://cloud.r-project.org/")
   GetISBI_data()
   library(imager)
   IMG_SIZE <- 168

